app/views/users/show.html.erb

```diff
@@ -0,0 +1,50 @@
+<% @title="My Blog | #{@user.name}" %>
+<div class='mt-3 d-flex align-items-center flex-column'>
+  <div class="card mb-3 border-primary" style="width: 680px;">
+    <div class="row g-0">
+      <div class="col-md-3">
+        <img src="<%= @user.photo %>" class="img-fluid rounded-start" alt="...">
+      </div>
+      <div class="col-md-4">
+        <div class="card-body">
+          <h2 class="card-title"><%= @user.name %></h2>
+        </div>
+        <div class='d-flex justify-content-end' style="width: 350px;">
+          <p class="card-text"><small class="text-muted">Number of posts: <%= @user.posts_counter %></small></p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="card mb-3 border-info" style="width: 680px;">
+    <div class='card-header bg-info'>Bio</div>
+    <div class="card-body">
+      <p class="card-text"><%= @user.bio %></p>
+    </div>
+  </div>
+  <% if @posts.empty? %>
+    <div class="card mb-3 border-danger" style="width: 540px;">
+      <div class="card-header bg-danger text-white">No Post Found</div>
+      <div class="card-body">
+        <h5 class="card-title text-danger">Please first submit a post then come back to this page thanks.</h5>
+      </div>
+    </div>
+  <% else %>
+    <% @posts.each do |post| %>
+      <%= link_to user_post_path(author_id:@user.id, id: post.id), class:'nav-link' do%>
+        <div class="card mb-3 border-success" style="width: 540px;">
+          <div class="card-header bg-success text-white"><%= post.title %></div>
+          <div class="card-body">
+            <h5 class="card-title"><%= post.title %></h5>
+            <p class="card-text"><%= post.text.slice(0,70)%> ...</p>
+          </div>
+          <div class='card-footer d-flex gap-2 justify-content-end'>
+            <p class="card-text"><small class="text-muted">Comments: <%= post.comments_counter %></small></p>
+            |
+            <p class="card-text"><small class="text-muted">Likes: <%= post.likes_counter %></small></p>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  <% end %>
+  <button class='btn btn-primary mb-3'>Show all Posts</button>
```

Comment:
- [ ] [OPTIONAL] Kindly make the `See all Posts` button redirect the user to the `Post#index` for a better user experience.